src/ssb/index.js

```diff
@@ -65,7 +77,15 @@ let clientHandle;
  */
 const attemptConnection = () =>
   new Promise((resolve, reject) => {
-    connect({ remote })
+    const originalConnect = process.env.OASIS_TEST
+      ? new Promise((resolve, reject) =>
+          reject({
+            message: "could not connect to sbot",
+          })
+        )
+      : connect({ remote });
+
+    originalConnect
```

Comment:
This is the bit meant to keep it from connecting to Patchwork/etc -- usually it does:

- try connection to existing server
  - success: keep the connection
  - fail: make a new one
    - success: connect to it
    - fail: crash the server

This change automatically fails the "try connection to existing server" step during tests, which means that it tries to make a new server. If it can make a new server then we know that Patchwork/etc aren't running on that port, and we can run the tests. If it *can't* make a new server, then Patchwork is probably running and we want to crash loudly.